benchmarks/basic/README.md

```diff
@@ -22,4 +22,6 @@ while `cpp-driver` takes just the ip address and assumes the port is `9042`
 Normally when the `selects` workload is used the driver will crate a keyspace, table and insert the values to select.
 When this option is disabled the benchmark assumes this has already been done. This can be achieved by first running
 the driver with `inserts` workload.  
-(Not enabled by default)
\ No newline at end of file
+(Not enabled by default)
+
+Abbreviated versions of command line arguments do not work with `gocql` benchmark.
```

Comment:
Maybe `Short versions` would be better? Abbreviated is a difficult word

Reply:
abbreviated is a beautiful word, wont fix